Cargo.toml

```diff
@@ -11,11 +11,27 @@ features = "0.10.0"
 openmls_traits = "0.1.0"
 serde = { version = "1.0.147", features = ["derive"] }
 tls_codec = { version = "0.2.0", features = ["derive", "serde", "mls"] }
+openmls = { path = "../../openmls/openmls/openmls" }
+openmls_rust_crypto = { version = "0.1.0", path = "../../openmls/openmls/openmls_rust_crypto" }
```

Comment:
These look like local paths

Reply:
Sorry about that. OpenMLS needs a few changes to its interface before we can use it, so I used a local version for development. The changes are now on a branch called `konrad/public_group_interface`.